### DISCUSSION TITLE
Content area width does not appear to work

### DISCUSSION THREAD

**Discussion by ronwildt** [squidfunk/mkdocs-material#4762] [Q&A] [answered]
Testing out expanding the content area to utilize the full width of the screen, documented [here](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width), does NOT appear to work?

Any guidance would be appreciated as this is constraining the use of mkdocs-material for our team.  Thank you!

**Comment by squidfunk** [comment, marked as answer]
Works for me – you can test it on the website using the inspector (see bottom right corner):

https://user-images.githubusercontent.com/932156/207704146-ec1be0a2-7a0d-4c50-8450-98a64497a7c8.mp4

**Reply by ronwildt** [reply]
I searched for 'inspector' on your website but did not find it?  Is this a third party tool or something you have built into the website?

The code you show below the Setting up navigation page looks to be HTML?  The solution above appears to only need 2 changes, one to extra.css and mkdocs.yml.  I have made both changes, saved the files and stopped and restarted 'mkdocs serve' to preview locally but no change?

From the short video clip above, it looks like you are simply increasing the font size of the web site (Command + / -) which works on any website...please let me know how to replicate as we want to test and roll this out next week if possible.
Thank you! :)

**Reply by squidfunk** [reply]
> I searched for 'inspector' on your website but did not find it? Is this a third party tool or something you have built into the website?

I was talking about the Chrome web inspector, see here for a [short tutorial](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/debugging/chrome-inspector.html).

> The solution above appears to only need 2 changes, one to extra.css and mkdocs.yml. I have made both changes, saved the files and stopped and restarted 'mkdocs serve' to preview locally but no change?

I'm not sure what you're doing wrong. As I said, I've verified that it works. I recommend consulting with somebody on your team who knows basic CSS. If you follow the guide on our documentation that explains [how to adjust content area width](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width), it will work.

**Comment by ronwildt** [comment]
Thank you for the Chrome web inspector link, makes more sense now!  I was able to figure out the issue with our CSS code with some team help.  Thanks again and have a great week & super Holiday Season! :)

**Reply by squidfunk** [reply]
Could you elaborate what you did wrong? This would be of great value for future users struggling with similar problems 😊
